addons/l10n_uy/tests/test_doc_types.py

```diff
@@ -0,0 +1,60 @@
+from odoo import Command
+from odoo.tests.common import tagged
+
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+
+
+@tagged("-at_install", "post_install", "post_install_l10n")
+class TestDocTypes(AccountTestInvoicingCommon):
+
+    @classmethod
+    def setUpClass(cls, chart_template_ref='uy'):
+        super().setUpClass(chart_template_ref=chart_template_ref)
+
+        service_vat_22 = cls.env["product.product"].create({
+            "name": "Virtual Home Staging (VAT 22)",
+            "list_price": 38.25,
+            "standard_price": 45.5,
+            "type": "service",
+            "default_code": "VAT 22",
+        })
+
+        cls.invoice = cls.env["account.move"].with_context(
+            default_move_type="out_invoice").create({
+                "partner_id": cls.env["res.partner"].create({"name": "test partner UY"}).id,
+                "move_type": "out_invoice",
+                "l10n_latam_document_type_id": cls.env.ref("l10n_uy.dc_e_inv_exp").id,
+                "invoice_line_ids": [Command.create({
+                    "product_id": service_vat_22.id,
+                    "quantity": 1.0,
+                    "price_unit": 100.0,
+                    "display_type": "product",
+                })]
+            })
+        cls.invoice.action_post()
+
+    def test_credit_note(self):
+        self.assertEqual(self.invoice.l10n_latam_document_type_id.code, "121", "Not Export e-Invoice")
+
+        refund_wizard = self.env["account.move.reversal"].with_context({
+            "active_ids": [self.invoice.id], "active_model": "account.move"}).create({
+                "reason": "Mercadería defectuosa",
+                "journal_id": self.invoice.journal_id.id
+            })
```

Comment:
we usually indent like that:
```suggestion
        move_reversal = self.env['account.move.reversal']\
            .with_context(active_model="account.move", active_ids=self.invoice.ids)\
            .create({
                ...
            })